packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/ReviewWorkflowsStage/ReviewWorkflowsStage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from '@strapi/design-system';
+
+export function ReviewWorkflowsStage({ name }) {
+  return (
+    <Typography fontWeight="bold" textColor="neutral700">
```

Comment:
The font weight doesn't look quite right to me
```suggestion
    <Typography fontWeight="regular" textColor="neutral700">
```

Reply:
Good catch!